src/lemon/config.h

```diff
@@ -0,0 +1,24 @@
+#define LEMON_VERSION "1.3.1"
+#define LEMON_HAVE_LONG_LONG 1
+
+//Define when you have LEMON LP Solvers
```

Comment:
Remove this comment. Even if we need to support LP interfaces in the future, this will not be the way we do it (we will have `setup.py` generate `config.h` based on command-line flags).